packages/shared/src/components/feed/LeanFeed.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import LeanFeedItemComponent from './LeanFeedItemComponent';
+import { useActiveFeedContext } from '../../contexts';
+import { FeedItem } from '../../hooks/useFeed';
+
+function LeanFeed({ items }: { items: FeedItem[] }): ReactElement {
+  const { feedRef } = useActiveFeedContext();
+
+  return (
+    <div className="grid grid-cols-2 gap-4" ref={feedRef}>
```

Comment:
If this is to be a generic component, I suggest accepting classes from the outside and set it with default value

Reply:
Good point indeed, most likely will take default "feed settings" and can get overwritten from outside.